src/decision/location_selection.jl

```diff
@@ -185,8 +187,18 @@ function rank_locations(
             ranks_store[locations=At(selected_seed_ranks), intervention=At(:seed), scenarios=scen_idx] .= 1:length(selected_seed_ranks)
         end
 
+        fog_decision_mat = decision_matrix(
```

Comment:
Same comment as above. This will crash if the location inputs don't contain viable seeding locations.